scripts/createConstants.js

```diff
@@ -0,0 +1,60 @@
+const { readFile, writeFileSync } = require('fs')
+const path = require('path')
+
+
+const isMainnet = process.argv.includes('mainnet')
+const isHolesky = process.argv.includes('holesky')
+
+let configName = null
+
+if (isMainnet) {
+  configName = 'mainnet'
+}
+
+if (isHolesky) {
+  configName = 'holesky'
+}
+
+if (!configName) {
+  throw new Error('Network is not supported')
+}
+
+const zeroAddress = '0x0000000000000000000000000000000000000000'
+const configPath = path.resolve(__dirname, '..', 'src', 'config', `${configName}.json`)
+const resultPath = path.resolve(__dirname, '..', 'src', 'helpers', `constants.ts`)
```

Comment:
I suggest we remove this file from the repo and add it to `.gitignore` so that it won't get accidentally committed.